monkey/monkey_island/cc/ui/src/components/configuration-components/ConfigurationTabs.js

```diff
@@ -1,23 +1,23 @@
 const CONFIGURATION_TABS = {
-  BASIC: 'basic',
-  BASIC_NETWORK: 'basic_network',
-  RANSOMWARE: 'ransomware',
-  MONKEY: 'monkey',
-  INTERNAL: 'internal'
+  GENERAL: 'general',
```

Comment:
We either don't need this tab or it should be something called "Advanced" and go last, because most users won't be changing their "keep tunnel open time". Neither they will understand what it means

Reply:
Probably you are right, but this schema will be reworked later again, so the order is not really important now.